nifi-framework-bundle/nifi-framework/nifi-headless-server/src/main/java/org/apache/nifi/headless/HeadlessNiFiServer.java

```diff
@@ -195,6 +196,10 @@ public void preDestruction() throws AuthorizerDestructionException {
         }
     }
 
+    protected void validateFlow() {
+        logger.info("Flow validated successfully");
```

Comment:
Shouldn't it be better to just leave this empty or provide a more generic message stating "Flow Validation was not implemented" or something similar. Just asking because if this is not overridden then no validation is actually executed. What do you think?

Reply:
Thanks for checking the PR. Yes it makes sense, updated the log message